client/src/utils/calendar-helpers.js

```diff
@@ -80,9 +90,50 @@ const generateCalendarMonth = (monthIndex, year) => {
   return null;
 };
 
+const calculateDateTo = (dateFrom) => {
+  let dateTo = new Date(dateFrom.getFullYear(), dateFrom.getMonth(), dateFrom.getDate() + 61);
+  return dateTo;
+}
+const formatDate = (date) => {
+  if (date instanceof Date) {
+    let month = date.getMonth();
+    let day = date.getDate();
+    let year = date.getFullYear();
+    
+    return `${month + 1}/${day}/${year}`;
```

Comment:
I think date.toLocaleDateString() can help here

Reply:
maybe for next refactor

Reply:
Will do!